patreon/schemas/campaign_spec.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+
+from patreon.schemas import campaign
+
+
+@pytest.fixture
+def attributes():
+    return [
+        'summary',
+        'creation_name',
+        'pay_per_name',
+        'one_liner',
+        'main_video_embed',
+        'main_video_url',
+        'image_small_url',
+        'image_url',
+        'thanks_video_url',
+        'thanks_embed',
+        'thanks_msg',
+        'is_monthly',
+        'is_nsfw',
+        'is_charged_immediately',
+        'is_plural',
+        'created_at',
+        'published_at',
+        'pledge_url',
+        'pledge_sum',
+        'patron_count',
+        'creation_count',
+        'outstanding_payment_amount_cents',
+    ]
+
+
+@pytest.fixture
+def relationships():
+    return [
+        'rewards',
+        'creator',
+        'goals',
+        'pledges',
+        'current_user_pledge',
+        'post_aggregation',
+        'categories',
+        'preview_token',
+    ]
+
+
+def test_schema_attributes_are_properly_formatted(attributes):
+    for attribute_name in attributes:
+        value = getattr(campaign.Attributes, attribute_name, None)
+        assert value is not None and value is attribute_name
+
+
+def test_schema_relationships_are_properly_formatted(relationships):
+    for relationship_name in relationships:
+        value = getattr(campaign.Relationships, relationship_name, None)
+        assert value is not None and value is relationship_name
+
+
+def test_schema_has_expected_default_attributes():
```

Comment:
this test probably isn't necessary. We should be testing business logic not declarations.